src/server/new-request-handlers/commits.js

```diff
@@ -1,72 +1,28 @@
 /*jshint loopfunc: true */
 "use strict";
 
-var pool = require('../db/index.js');
+var users_dashboards = require('../queries/users_dashboards.js');
+var diffs = require('../queries/diffs.js');
 
 module.exports = {
   handlePost: function (req, res, next) {
-    var dashboards_id = req.body.dashboards_id;
+    var dashboards_signature_hash = req.body.dashboards_signature_hash;
     var diffs = req.body.diffs;
```

Comment:
We'll need to rename diffs to something different, since we now have a diffs var for the queries/diffs module.